config.yaml

```diff
@@ -92,13 +116,17 @@ submodels:
         - /g/data/vk83/configurations/inputs/access-esm1p5/modern/share/coupler/remapping_weights/global.oi_1deg.a_N96/2020.05.19/rmp_um1t_to_cice_CONSERV_FRACNNEI.nc
         - /g/data/vk83/configurations/inputs/access-esm1p5/modern/share/coupler/remapping_weights/global.oi_1deg.a_N96/2020.05.19/rmp_cice_to_um1v_CONSERV_FRACNNEI.nc
 
+# Collation
 collate:
-   exe: /g/data/access/payu/access-esm/bin/mppnccombine
+   exe: mppnccombine.spack
    restart: true
    mem: 4GB
+   walltime: 1:00:00
+   mpi: true
```

Comment:
Do we need to swap this one to `mpi: false` as with the `pre-industrial` configuration?

Reply:
Confirmed that this is the case - yes, we do